plugins/woocommerce/client/admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -237,100 +304,126 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
-									<CardLink />
-								</h2>
-								{ showVendorLoading && (
-									<p className="woocommerce-marketplace__product-card__vendor-details">
-										<span className="woocommerce-marketplace__product-card__vendor" />
-									</p>
+					<div className="woocommerce-marketplace__product-card__meta">
+						<h2 className="woocommerce-marketplace__product-card__title">
+							<CardLink />
+						</h2>
+						{ showVendorLoading && (
+							<p className="woocommerce-marketplace__product-card__vendor-details">
+								<span className="woocommerce-marketplace__product-card__vendor" />
+							</p>
+						) }
+						{ showVendor && (
+							<p className="woocommerce-marketplace__product-card__vendor-details">
+								{ productVendor && (
+									<span className="woocommerce-marketplace__product-card__vendor">
+										<span>
+											{ __( 'By ', 'woocommerce' ) }
+										</span>
+										{ productVendor }
+									</span>
+								) }
+								{ productVendor && isSponsored() && (
+									<span
+										aria-hidden="true"
+										className="woocommerce-marketplace__product-card__vendor-details__separator"
+									>
+										·
+									</span>
 								) }
-								{ showVendor && (
-									<p className="woocommerce-marketplace__product-card__vendor-details">
-										{ productVendor && (
-											<span className="woocommerce-marketplace__product-card__vendor">
-												<span>
-													{ __(
-														'By ',
-														'woocommerce'
-													) }
-												</span>
-												{ productVendor }
-											</span>
-										) }
-										{ productVendor && isSponsored() && (
-											<span
-												aria-hidden="true"
-												className="woocommerce-marketplace__product-card__vendor-details__separator"
-											>
-												·
-											</span>
-										) }
-										{ isSponsored() && (
-											<span className="woocommerce-marketplace__product-card__sponsored-label">
-												{ __(
-													'Sponsored',
-													'woocommerce'
-												) }
-											</span>
-										) }
-									</p>
+								{ isSponsored() && (
+									<span className="woocommerce-marketplace__product-card__sponsored-label">
+										{ __( 'Sponsored', 'woocommerce' ) }
+									</span>
 								) }
-								{ isCompact && footer }
-							</div>
-						</div>
+							</p>
+						) }
+						{ isCompact && footer }
 					</div>
-					{ showDescription && (
-						<p className="woocommerce-marketplace__product-card__description">
-							{ ! isLoading && decodedDescription }
-						</p>
-					) }
-					{ ! isCompact && footer }
 				</div>
+			</div>
+			{ showDescription && (
+				<p className="woocommerce-marketplace__product-card__description">
+					{ ! isLoading && decodedDescription }
+				</p>
+			) }
+			{ ! isCompact && footer }
+		</div>
+	);
+
+	const CardWrapper = () => {
+		if ( ! isLoading ) {
```

Comment:
I'm not a huge fan of negative conditions. I'd probably change it to be:

```js
if ( isLoading ) {
    return isBusinessService ? <BusinessService /> : cardContent;
}

return (
 <div ....
);

Reply:
Makes sense. Done in ae04f82f1e